src/RoleObjects/MultiLineTextBoxData.js

```diff
@@ -25,6 +28,20 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
     throw new Error(`${this.role} cannot have children`);
   }
 
+  /**
+   * @inheritdoc
+   */
+  set enableKeyEvents(enable) {
+    super.enableKeyEvents = enable;
```

Comment:
In the base class (https://github.com/CurriculumAssociates/createjs-accessibility/blob/main/src/RoleObjects/AccessibilityObject.ts#L270-L273) if the setter is being called with false, it will clear both the key down and up listeners.  Since you want to have your keyup one be always bound, you'll probably want https://github.com/CurriculumAssociates/createjs-accessibility/blob/main/src/RoleObjects/GridData.js#L18-L19 here but for key up.

Reply:
Yess, That makes sense! Thankyou...